packages/datadog-plugin-graphql/test/index.spec.js

```diff
@@ -1545,5 +1545,121 @@ describe('Plugin', () => {
         })
       })
     })
+
+    withVersions(plugin, 'graphql', '>10.0.0', toolsVersion => {
```

Comment:
Multiple `withVersions` blocks will make the output confusing. This can be replaced by adding additional version ranges in `externals.json`, and using `semver` to limit specific tests to a subset of all the ranges.